File: app/src/main/java/de/techlung/android/mortalityday/util/Toaster.java
package de.techlung.android.mortalityday.util;

import android.widget.Toast;

import de.techlung.android.mortalityday.MainActivity;
import de.techlung.android.mortalityday.R;

public class Toaster {

    public static void show(String message) {
        if (MainActivity.getInstance() != null) {
            Toast.makeText(MainActivity.getInstance(), message, Toast.LENGTH_LONG).show();
        }
    }

    public static void showNetworkError() {
        if (MainActivity.getInstance() != null) {
            Toast.makeText(MainActivity.getInstance(), MainActivity.getInstance().getString(R.string.error_no_network), Toast.LENGTH_SHORT).show();
        }
    }
}

File: app/src/main/java/com/techlung/android/mortalityday/util/Toaster.kt
package com.techlung.android.mortalityday.util

import android.content.Context
import android.widget.Toast

object Toaster {
    fun show(message: String, context: Context?) {
        if (context != null) {
            Toast.makeText(context, message, Toast.LENGTH_LONG).show()
        }
    }
}
